### DISCUSSION TITLE
Synthesise abbreviations in YourTTS

### DISCUSSION THREAD

**Discussion by anusha-potnuru** [coqui-ai/TTS#2545] [General Q&A]
I am trying to synthesise abbreviations i.e each character individually, but both in zero shot and fine-tuned models the audios for the characters are coming out like mumbling or partial sounds.

Is there any solution for this?

**Comment by erogol** [comment]
not any built in solution. You need to expand them before giving it to the model

**Reply by harshvardhan-truefan** [reply]
Hi @erogol,
I'm getting the same issue after fine-tuning YourTTS model on my dataset. I wanted to know what do you mean by "expand them before giving to the model" above ? 

Really appreciate your help ! Look forward to hearing from you. 

Regards,
Harsh

**Reply by eginhard** [reply]
To pronounce "ARD" as individual letters, you can expand it to "A.R.D.". But this likely still won't result in the correct pronunciation because of Coqui's preprocessing. You can apply this fix for that, but as I mentioned there I don't know if this has other side effects: https://github.com/coqui-ai/TTS/issues/2619#issuecomment-1550326290
